### DISCUSSION TITLE
How to revert to older version of PHP ?

### DISCUSSION THREAD

**Discussion by freebrowser1** [termux/termux-packages#21300] [Q&A]
Since PHP 8.3.10, the module gd is not included anymore in PHP, which did until 8.3.8. But I use gd, so I have made a bug report #21262 on this and in the meantime I want to revert PHP to 8.3.8 so I can use the gd plugin.
How can I revert to a (slightly) older version of a package ?

**Comment by anothermomo** [comment]
You can now install another Linux system, `fedora`, on Termux using `PRoot`, then install `Remi` on this Linux system. Remi provides multiple different versions of PHP for download. I haven't found the version 8.3.8 on Remi, but older versions may be helpful.
